packages/gatsby/cache-dir/loader.js

```diff
@@ -236,6 +236,9 @@ const queue = {
       prefetchResource(page.componentChunkName)
     }
 
+    // Tell plugins the path has been successfully prefetched
+    apiRunner(`onPostPrefetchPathname`, { pathname: path })
```

Comment:
it this point we don't know if resources are prefetched yet - only that we started prefetching already I think?